recipes/hipace/meta.yaml

```diff
@@ -0,0 +1,84 @@
+{% set name = "hipace" %}
+{% set version = "23.03" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Hi-PACE/hipace/archive/v{{ version }}.tar.gz
+  sha256: f32104af1e7ca33b7ef8a3fd7964ee62db13a68a00af305d4ace5d0e86e2efb2
+
+build:
+  number: 0
+  run_exports:
+    # releases are not (yet) compatible
+    - {{ pin_subpackage('hipace', max_pin='x.x') }}
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
```

Comment:
Python in the build and not host requirements implies that you are just using python as build tool and that your package does not link to or have API requirements of the python version.

If this is the case, then you don't need to build against multiple python versions. You should probably be ignoring run_exports from python.

If this is not the case, then python should also be listed in the host section.

Reply:
This was due to the `ctest` requiring Python scripts. Since we do not run ctest anymore in the build phase, we can absolutely simplify the build combinations now :tada: